### PR TITLE
Step preview layout

### LINKED ISSUE #299: Improve live preview
_As a user
I want to see, when possible, the outcome of my changes alongside my answers
So that I can be confident my answers fit into the context of the final consent form_

## Description

When there is enough space, the preview can be placed alongside the form so that it can be seen more easily by the user.

## Acceptance Criteria

- [ ] The width of the product shouldn't be constrained
- [ ] When there is enough space, the preview should be moved alongside the form
- [ ] Otherwise, it should stay below the form
- [ ] Ensure the line length of text doesn't exceed 95 characters

### PR DESCRIPTION
**Pull request by joelcarr**
Closes #299.

### CODE REVIEW

File: app/views/research_sessions/questions/methodologies.html.erb [hunk 1]
```diff
@@ -3,76 +3,80 @@
...
-          url: edit_link_for(:researcher_name)
-        ),
-        c(
-          'Output',
-          text: @research_session.researcher_job_title,
-          field: 'researcher_job_title',
-          url: edit_link_for(:researcher_job_title)
-        )
-      ] %>
+      <%= render partial: 'submit_step' %>
```
Comment (kittysquee) [inline app/views/research_sessions/questions/methodologies.html.erb]: This can be written as <%= render 'submit_step' %>

**Comment by kittysquee** [comment]
screen shot 2018-06-13 at 14 17 12

The text at the top seems misaligned and the preview seems misaligned with the input.

File: app/views/research_sessions/questions/methodologies.html.erb (hunk 1 above)
Comment (joelcarr) [reply app/views/research_sessions/questions/methodologies.html.erb]: Could this be part of the Ruby refactor?

Comment (kittysquee) [reply app/views/research_sessions/questions/methodologies.html.erb]: Certainly could be 👍

**Comment by joelcarr** [comment]
@kittysquee We could apply negative margin to the top of the preview to try and make the text align. What do you think @kelliewellingtons?

**Comment by kelliewellingtons** [comment]
@kittysquee @joelcarr I'm satisfied with the layout and implementation in this PR. 

Any further work would potentially be made redundant by future tickets in which the content is being overhauled.

**Pull request merged**